controllers/core/configmap_controller.go

```diff
@@ -67,11 +68,27 @@ func (r *ConfigMapReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	// Check if the flag value has changed
 	newWinIPAMEnabledCond := r.Condition.IsWindowsIPAMEnabled()
 
+	var isIPAMFlagUpdated bool
 	if r.curWinIPAMEnabledCond != newWinIPAMEnabledCond {
 		r.curWinIPAMEnabledCond = newWinIPAMEnabledCond
 		logger.Info("updated configmap", config.EnableWindowsIPAMKey, r.curWinIPAMEnabledCond)
 
-		// Flag is updated, update all nodes
+		isIPAMFlagUpdated = true
+	}
+
+	// Check if the prefix delegation flag has changed
+	newWinPrefixDelegationEnabledCond := r.Condition.IsWindowsPrefixDelegationEnabled()
+
+	var isPrefixFlagUpdated bool
+	if r.curWinPrefixDelegationEnabledCond != newWinPrefixDelegationEnabledCond {
+		r.curWinPrefixDelegationEnabledCond = newWinPrefixDelegationEnabledCond
+		logger.Info("updated configmap", config.EnableWindowsPrefixDelegationKey, r.curWinPrefixDelegationEnabledCond)
+
+		isPrefixFlagUpdated = true
+	}
+
+	// Flag is updated, update all nodes
+	if isIPAMFlagUpdated || isPrefixFlagUpdated {
```

Comment:
`OR` check makes sense to me. Please check the comment from the other PR (#205). I am thinking PrefixFlag should be able to override IPAMFlag.

Reply:
Thanks for making note of it. I'll resolve this comment for now since PM decides to not have override behavior for PD flag.